ballista/rust/core/src/serde/physical_plan/from_proto.rs

```diff
@@ -81,6 +83,41 @@ impl TryFrom<&protobuf::PhysicalExprNode> for Arc<dyn PhysicalExpr> {
                         .to_owned(),
                 ));
             }
+            ExprType::ScalarUdfProtoExpr(e) => {
+                if let Some(udf_plugin_manager) = get_udf_plugin_manager("") {
```

Comment:
This should be resolved from the `SessionContext` right?

Reply:
I added TODO and I plan to finish physical plan serialization and deserialization through SessionContext in the next PR.